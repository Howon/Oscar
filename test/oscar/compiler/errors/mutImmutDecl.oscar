message testNow()


actor Tester() {
  mut int x = 1;
  int x = 2;

  receive = {
    | testNow() => {}
      }
    | die() => {
      }
}

def main() => unit = {
  actor<Tester> tester = spawn actor<Tester>();
  message<testNow>() |> tester;
  message<die>() |> tester;
}
