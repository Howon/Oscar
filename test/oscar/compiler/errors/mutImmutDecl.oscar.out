x declared as both mutable and immutable